server/lib/routes/get-index.js

```diff
@@ -2,6 +2,20 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+var activityEvent = require('../activity-event');
+var crypto = require('crypto');
+
+var activityEventQueryParams = [
+  'context',
+  'entrypoint',
```

Comment:
They're not important for the new KPIs but it also can't do any harm to log them in case they're wanted in the future.

It would mean more stuff needs to be added to the auth server API in https://github.com/mozilla/fxa-auth-server/pull/1213 and https://github.com/mozilla/fxa-js-client/pull/187 though, so I'll defer to @rfk on this. Ryan?